boards/sensebox_samd21/doc.txt

```diff
@@ -53,4 +53,12 @@ The pin configuration for the mSD-Bee module radio on the
 *boards/sensebox/include/sdcard_spi_params.h** file assumes that it will be
 connected to the XBEE2 port. If a different configuraion is needed please
 modify this file.
- */
\ No newline at end of file
+
+#### Limitations
+There is only one interrupt pin on the XBEE headers. This means that the
+interrupt pins from the LoRa Bee are multiplexed. This has the following
```

Comment:
Maybe specify that DIO0 and DIO1 are the pins connected to an OR gate?

Reply:
I've pushed a change. Are you ok with this?

If no more comments, I'll squash and rebase.